homework/content.js

```diff
@@ -1,21 +1,26 @@
 'use strict';
 
-(function() {
+(function(exports) {
   var _wrapper = document.querySelector('#note-content-wrapper');
 
-  function start() {
-    window.addEventListener('note-open', function(event) {
-      var note = event.detail;
-      resetWrapper();
-      drawNote(note);
-    });
+  var ContentManager = function() {
+  this._wrapper = document.querySelector('#note-content-wrapper');
   }
 
-  function resetWrapper() {
-    _wrapper.innerHTML = '';
-  }
+  ContentManager.prototype = {
+  start() {
+    window.addEventListener('note-open', (function(event) {
```

Comment:
Rather than binding on an anonymous function, you need to bind the `this` via set it as the listener of `addEventListener`.